onnxruntime/core/providers/webnn/builders/impl/reduction_op_builder.cc

```diff
@@ -117,6 +133,13 @@ bool ReductionOpBuilder::IsOpSupportedImpl(const InitializedTensorSet& /* initia
   if (!GetShape(*input_defs[0], input_shape, logger))
     return false;
 
+  const auto& op_type = node.OpType();
+  // If the optional input 'axes' is provided, it must be an initializer.
+  if (input_defs.size() > 1 && !Contains(initializers, input_defs[1]->Name())) {
+    LOGS(logger, VERBOSE) << "Input axes of " << op_type << " must be known";
```

Comment:
Clearer error message? e.g. Input axes must be a constant

Reply:
Sure, done.